### DISCUSSION TITLE
Wordpress speed optimization

### DISCUSSION THREAD

**Discussion by floorajones** [community/community#53789] [Programming Help]
### Body

Hi,
Could anyone help me with improving the speed of my WordPress [website on the tattoo niche](https://tatglam.com/)? Currently, the site is experiencing speed issues on mobile devices, such as Total Blocking Time and Largest Contentful Paint. Can anyone advise me on how to improve this issue? I am currently using WP Rocket and Perfmatters plugins, but these plugins are unable to resolve the problem.

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by hrishikesh-kocharekar-dev** [comment]
if you're using WP Rocket and Perfmatters plugins, they don't seem to be resolving the issue. Improving website speed can be a complex process, but some potential solutions could include optimizing images, using a content delivery network (CDN), and minimizing JavaScript and CSS files.

**Comment by floorajones** [comment]
I did all of these things except for using a CDN. Is there anything else that I should try to improve my site's speed?

**Comment by theaminulai** [comment]
Upon a comprehensive assessment of your website's performance.

To address this concern, I recommend implementing [quicklink](https://github.com/GoogleChromeLabs/quicklink). Quicklink is a lightweight solution designed to enhance user experience by preloading links within the viewport, thereby optimizing the loading of subsequent pages. By incorporating quicklink, you can notably improve the overall speed and responsiveness of your website.

Feel free to integrate this solution, and if you have any questions or require further assistance in optimizing your website, don't hesitate to reach out.

**Comment by HiWebDev0026** [comment]
Hi there! I can help you.

To improve the speed of your WordPress website in the tattoo niche, especially on mobile devices, where you are experiencing Total Blocking Time and Largest Contentful Paint issues, there are several actions you can take:

Image Optimization: Compress and optimize images to reduce file sizes without sacrificing quality. You can use plugins like Smush or EWWW Image Optimizer for this purpose.

Minify CSS and JavaScript: Minification reduces the file size of CSS, JavaScript, and HTML by removing unnecessary characters such as spaces, comments, and formatting.

Content Delivery Network (CDN): Utilize a CDN to serve your content from servers closer to the user's location, reducing latency and improving loading times.

Caching: Ensure that your caching plugins, such as WP Rocket, are configured optimally to store static versions of your site's content, reducing the need to generate it with each visit.

Reduce Redirects: Minimize the number of redirects as they add additional time to the loading process.

Quality Web Hosting: Consider upgrading to a web host optimized for WordPress sites, as the hosting infrastructure can have a significant impact on loading times.

Content Prioritization: Prioritize visible content and critical resources to mitigate Largest Contentful Paint issues.

Code Cleanup: Review your website's code to eliminate unnecessary elements and improve its efficiency.

It's important to monitor the impact of each change and consider seeking professional assistance if the issues persist.

**Comment by junaid9940** [comment]
1) convert all images into .webp format 
2) use litespeed cache plugin

**Comment by chshydnews** [comment]
WordPress speed optimization helps a website load faster, improve SEO, and give users a smooth experience. CHS HYD News also keeps its site fast and clean for quick updates on auto, finance, tech, weather, entertainment, and gaming: https://chshyd.in/
